docs/machine_learning/3_evaluation.md

```diff
@@ -0,0 +1,195 @@
+# Model Evaluation
+
+Before deploying a model to be consumed by several applications,
+we need to know what is the expected performance that it will achieve.
+After training a model using the collected and transformed dataset,
+concrete metrics have to be calculated so we can have a vision of
+how well the algorithm can model the data that we have on hand.
+
+Depending on the type of the problem (supervised, unsupervised,
+classification, regression), there are different metrics
+to evaluate. In supervised learning, we have available target values
+that we can use to compare to what was predicted by the model.
+However, the metrics are different between classification and regression problems
+since in the first one the target variable is not continuous.
+
+Another topic is worth mentioning, and that can lead to misleading
+conclusions is what data should be used to evaluate. If we train
+the model with the entire dataset, make predictions over the same dataset,
+and compare these predictions with the actual values, we are not
+verifying how the model behaves with unseen data samples. That's why
+before speaking about evaluation metrics, we have to jump into
+data split strategies, to be able to separate the data into
+different sets depending on their use.
+
+## Data Split Strategies
+
+A data split strategy is a way of splitting the data
+into different subsets, so each subset is used with a different goal.
+This will prevent us from using the entire dataset for training, tuning
+, and evaluating which can give false conclusions.
+
+The most used strategies are:
+
+- train-test split
+- cross-validation
+- leave one out
+
+The idea behind the train-test split, as the name suggests, is to split
```

Comment:
You could add some headers for this with ###.

One before the explanation of each of the bullet points.